Makefile.sparcv8leon3

```diff
@@ -0,0 +1,36 @@
+#
+# Makefile for libphoenix
+#
+# SPARCv8 LEON3 options
+#
+# Copyright 2022 Phoenix Systems
+#
+# %LICENSE%
+#
+
+CROSS ?= sparc-phoenix-
+
+CC = $(CROSS)gcc
+
+OLVL ?= -O2
+CFLAGS += $(OLVL)
+CFLAGS += -Wall -Wstrict-prototypes -g\
+	-fno-builtin-malloc -fdata-sections -ffunction-sections -mcpu=leon3
+
+AR = $(CROSS)ar
+ARFLAGS = -r
+
+LD = $(CROSS)ld
+LDFLAGS := --gc-sections
+GCCLIB := $(shell $(CC) $(CFLAGS) -print-libgcc-file-name)
+CRTBEGIN := $(shell $(CC) $(CFLAGS) -print-file-name=crtbegin.o)
+CRTEND := $(shell $(CC) $(CFLAGS) -print-file-name=crtend.o)
+PHOENIXLIB := $(shell $(CC) $(CFLAGS) -print-file-name=libphoenix.a)
+LDLIBS := $(PHOENIXLIB) $(GCCLIB) $(CRTBEGIN) $(CRTEND)
+
+OBJCOPY = $(CROSS)objcopy
+OBJDUMP = $(CROSS)objdump
+
+STRIP = $(CROSS)strip
+
+VADDR_KERNEL_INIT = 0x40000000
```

Comment:
Why not standard kernel virtual address 0xc0000000?

Reply:
Qemu for Leon3 doesn't emulate MMU, so I guess it has to be physical address.